pkg/scalers/http_clients.go

```diff
@@ -0,0 +1,20 @@
+package scalers
```

Comment:
Just a small nit, could you please move this file out of the `scalers` package, preferably somewhere under `pkg/util`?

Reply:
done in https://github.com/kedacore/keda/pull/1251/commits/833043a665b27a8909e3667fe016481efd288d78